source/wp-content/themes/wporg-showcase-2022/functions.php

```diff
@@ -316,3 +317,46 @@ function document_title( $parts ) {
 function document_title_separator( $title ) {
 	return '&#124;';
 }
+
+/**
+ * Add meta tags for richer social media integrations.
+ */
+function add_social_meta_tags() {
+	$default_image = get_stylesheet_directory_uri() . '/images/social-image.png';
+	$site_title    = function_exists( '\WordPressdotorg\site_brand' ) ? \WordPressdotorg\site_brand() : 'WordPress.org';
+	$og_fields = [
+		'og:title'       => wp_get_document_title(),
+		'og:description' => __( 'Discover inspiration in some of the most beautiful, best designed WordPress websites.', 'wporg' ),
+		'og:site_name'   => $site_title,
+		'og:type'        => 'website',
+		'og:url'         => home_url(),
```

Comment:
This should use a trailing-slashed URL:
```suggestion
		'og:url'         => home_url('/'),
```